src/RefractSourceMap.h

```diff
@@ -0,0 +1,73 @@
+//
+//  RefractSourceMap.h
+//  drafter
+//
+//  Created by Jiri Kratochvil on 31/07/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#ifndef DRAFTER_REFRACTSOURCEMAP_H
+#define DRAFTER_REFRACTSOURCEMAP_H
+
+#include "Serialize.h"
+
+namespace drafter {
+
+
+    refract::IElement* BytesRangeToRefract(const mdp::BytesRange& bytesRange);
+
+    template<typename T>
+    refract::IElement* SourceMapToRefract(const T& sourceMap) 
```

Comment:
What? Did you reply to the wrong comment? This is almost the same code as `SourceMap` functions in `RefractParseResult.*` files. So, I was asking you to move that code here and remove this part.